src/main/java/com/openlattice/authorization/EdmAuthorizationHelper.java

```diff
@@ -112,15 +112,99 @@ public EdmAuthorizationHelper( EdmManager edm, AuthorizationManager authz ) {
     }
 
     /**
-     * @see EdmAuthorizationHelper#getAuthorizedPropertiesOnEntitySets
+     * @see EdmAuthorizationHelper#getAuthorizedPropertiesByNormalEntitySets(EntitySet, EnumSet, Set)
      */
-    public Map<UUID, Map<UUID, PropertyType>> getAuthorizedPropertiesOnEntitySets(
-            Set<UUID> entitySetIds,
+    public Map<UUID, Map<UUID, PropertyType>> getAuthorizedPropertiesByNormalEntitySets(
```

Comment:
Maybe we should call them UnitaryEntitySets?

Reply:
We already started to call them normal entity sets, so unless there is a strong reason, I wouldn't change it again. (it's just getting too confusing)